app/controllers/mentor/applications_controller.rb

```diff
@@ -8,6 +8,20 @@ def show
     @comment     = application.find_or_initialize_comment_by(current_user)
   end
 
```

Comment:
We have two kind of 'comments' in the project: the polymorphic comments, like for projects etc, and the 'notes', like the notes in the supervisor dashboard. I think this is the note-kind, right? In that case, I'd prefer to reserve the term comment for the 'polymorphic' comments.